serveradmin/serverdb/models.py

```diff
@@ -48,7 +49,7 @@
 
 attribute_types = {
     'string': str,
-    'boolean': lambda x: x in ('1', 'True', 'true', 1, True),
+    'boolean': strtobool,
```

Comment:
Im really not sure, but wasnt there another location with something like that, too? Can't see it anymore here though.

Reply:
There is one more on `querybuilder.py`.  I will deal with that one by not casting but erroring out.  We first need to wait for everybody to upgrade though.  I put a `TODO` there for the time being.